.github/workflows/ci.yml

```diff
@@ -52,7 +52,7 @@ jobs:
       fail-fast: false
       matrix:
         parallelism: [5]
-        index: [0,1,2,3,4]
+        index: [1,2,3,4,5]
```

Comment:
I expect this will cause certain tests to be skipped cos this determines the PARALLEL_INDEX env var which is used to decide which tests to run in a given step

Reply:
Alrighty then, reverted it, now CI is finally green!